administration/logging.md

```diff
@@ -91,27 +91,29 @@ Note that the log levels set using the `log:set` commands are not persistent and
 
 ## Create Log Entries in Rules
 
-It is also possible to create own log entries in rules. This is especially useful for debugging purposes.
+There are times, especially when troubleshooting, when it can be helpful to create special rules that log specific variables or the State of Items.
```

Comment:
Not so sure about "create special rules". You normally add logging lines in a rule to troubleshoot it's execution. See for a random example by me: https://github.com/ThomDietrich/openhab-config/blob/master/rules/speedtest.rules#L33

Reply:
Yep - agree.  Will change text.